README.rst

```diff
@@ -77,6 +77,20 @@ Project goals
   ``xarray.DataArray`` for grids.
 
 
+Quickstart
+----------
+
+Installation
+++++++++++++
+
+Simple installation using `conda <https://docs.conda.io/projects/conda/en/latest/user-guide/index.html>`__
+(creates virtual environment and installs `pygmt` and required dependencies)::
+
+    conda create --name pygmt --channel conda-forge pygmt
```

Comment:
Question: Is it better to have `conda create ...` here (which also creates a new virtual environment), or just use `conda install` instead?

Reply:
I think for the quickest install instructions we could use conda install to avoid confusion about needing to activate the virtual environment.